requirements.txt

```diff
@@ -2,27 +2,26 @@ https://github.com/wazo-platform/wazo-lib-rest-client/archive/master.zip
 https://github.com/wazo-platform/wazo-auth-client/archive/master.zip
 https://github.com/wazo-platform/xivo-lib-python/archive/master.zip
 https://github.com/wazo-platform/xivo-bus/archive/master.zip
-alembic==1.0.0
-celery==4.4.6
-cheroot==6.5.4
-flask-cors==3.0.7
-flask-restful==0.3.7
-flask==1.0.2
+alembic==1.4.3
+celery==5.0.0
+cheroot==8.5.2
+flask-cors==3.0.9
+flask-restful==0.3.8
+flask==1.1.2
 httpx==0.7.2
```

Comment:
Why you didn't bump this version to 0.16.1?
https://packages.debian.org/bullseye/python3-httpx

Reply:
Oh sorry. I left it at the version we packaged. But yeah, it would be good to update. Should I bump it?

Reply:
It's a little more complicated than that, but thanks for the hint. :)